.gitignore

```diff
@@ -1 +1,2 @@
 /build/
+.idea/
```

Comment:
```suggestion
```
I would recommend to put that into a global git ignore file.

https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration.html#_core_excludesfile